back/.rubocop.yml

```diff
@@ -10,10 +14,134 @@ AllCops:
   - '**/*.builder'
   - '**/.*.*'
   - '**/bin/**/*'
+  - 'engines/ee/**/*'
 
 Style/StringLiterals:
   Enabled: true
 
+Layout/SpaceInsideHashLiteralBraces:
+  Enabled: true
+
+Layout/SpaceInsideBlockBraces:
+  Enabled: true
+
+Layout/SpaceAroundOperators:
+  Enabled: true
+
+Style/MethodDefParentheses:
+  Enabled: true
+
+Layout/SpaceBeforeBlockBraces:
+  Enabled: true
+
+Layout/SpaceAfterComma:
+  Enabled: true
+
+Layout/TrailingWhitespace:
+  Enabled: true
+  AllowInHeredoc: true
+
+Layout/EmptyLinesAroundBlockBody:
+  Enabled: true
+
+Style/BlockDelimiters:
+  Enabled: true
+
+Layout/EmptyLinesAroundClassBody:
+  Enabled: true
+
+Layout/HashAlignment:
+  # There are a lot of places where EnforcedColonStyle: table is expected.
```

Comment:
I played a bit with this rule once, and I think it was possible to support multiple configurations (for example `[key, table]`).